src/style.css

```diff
@@ -1 +1,70 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+  position: relative;
+}
+
+.logo__img {
+  position: absolute;
+  height: 40px;
+  width: 40px;
+  top: 10px;
+  left: 50px;
+}
+
+.header {
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav__list {
+  height: 60px;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  align-items: center;
+  margin: 0;
+}
+
+.nav__item {
+  display: inline-block;
+  padding: 0 10px;
+}
+
+.nav__item-last {
+  display: inline-block;
+  padding: 0 50px 0 10px;
+}
+
+.nav__link {
+  text-transform: uppercase;
+  text-decoration: none;
+  padding-top: 22.5px;
+  padding-bottom: 22.5px;
```

Comment:
Don't add paddings this way. Just make link as high as header. This should help with tests.